ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -313,12 +325,12 @@ $(document).ready(function() {
             var sequence = {};
             var sequence_identifier = $("#new-sequence-identifier-input").val();
             var authors_group_identifier = $("#ag-identifier-input").val();
-            $("#sequence-selector")
-            .append($("<option></option>")
-            .attr("value", sequence_identifier)
+            $("#sequence-selector");
+            .append($("<option></option>");
```

Comment:
Huh? I think the `.append` is supposed to act on the result of the `$("#sequence-selector")` so adding these semicolons breaks it!